metrics/prometheus.go

```diff
@@ -926,46 +931,66 @@ func (c *PrometheusCollector) collectContainersInfo(ch chan<- prometheus.Metric)
 			rawLabels[l] = struct{}{}
 		}
 	}
-	for _, container := range containers {
+
+	psReqOpt := v2.RequestOptions{
+		IdType: v2.TypeName,
+	}
+
+	for _, cont := range containers {
```

Comment:
nit: can we not rename the container variable?  it makes it harder to see code changes

Reply:
its conflicting with the container package that was imported. So need to rename the local variable.

Reply:
ah, ok.  It is fine then